README.md

```diff
@@ -1 +1,27 @@
-# esb-connector-cerner
\ No newline at end of file
+# Cerner EI Connector
+
+The Cerner [connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to access the Cerner FHIR APIs of [Cerner Millennium](https://fhir.cerner.com/millennium/dstu2/) EHR platform through WSO2 EI. Cerner connector currently supports the DSTU 2 Final (1.0.2) version of the FHIR standard.
```

Comment:
The Cerner [connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to access the Cerner FHIR APIs of the [Cerner Millennium](https://fhir.cerner.com/millennium/dstu2/) EHR platform through WSO2 Enterprise Integrator (WSO2 EI). Cerner connector currently supports the DSTU 2 Final (1.0.2) version of the FHIR standard.